Configurations/unix-Makefile.tmpl

```diff
@@ -531,9 +539,13 @@ build_all_generated: $(GENERATED_MANDATORY) $(GENERATED) build_docs
 	@echo "         then make will fail..."
 	@ : {- output_on() if $disabled{makedepend}; "" -}
 
-all: build_sw build_docs
+##@ Help
+.PHONY: help
+help: ## Show this help screen
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make <OPTIONS> \033[36m<target>\033[0m\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } END { printf "\nNote: This list is not all-inclusive\n" } ' $(MAKEFILE_LIST)
 
-test: tests
+##@ Testing
+test: tests ## Run tests
 {- dependmagic('tests'); -}: build_programs_nodep build_modules_nodep link-utils run_tests
```

Comment:
You have the same `dependmagic()` problem here...

Reply:
In this case, a just a blank line was removed.

Reply:
And the comment was appended.

Reply:
Oh, you're quite right!